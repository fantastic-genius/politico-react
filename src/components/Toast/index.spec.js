import React from 'react';
import { shallow } from 'enzyme';
import { Toast } from './index';

describe('<Toast />', () => {
  describe('Rendering', () => {
    it('should render Toast correctly', () => {
        const component = shallow(<Toast />);
        expect(component).toMatchSnapshot();
    });
  });
});
