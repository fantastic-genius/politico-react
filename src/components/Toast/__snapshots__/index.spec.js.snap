// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Toast /> Rendering should render Toast correctly 1`] = `
<h3
  className="toast-msg"
/>
`;
